scene/gui/tab_container.h

```diff
@@ -167,6 +171,12 @@ class TabContainer : public Container {
 	virtual Vector<int> get_allowed_size_flags_horizontal() const override;
 	virtual Vector<int> get_allowed_size_flags_vertical() const override;
 
+	void set_input_rollover(bool p_enabled);
+	bool get_input_rollover() const;
+
+	void set_tab_bar_focus_mode(FocusMode p_focus_mode);
+	FocusMode get_tab_bar_focus_mode() const;
+
```

Comment:
Should be next to related methods like `set/get_tab_alignment`. Same for the cpp file.